packages/lab/stories/tooltip.stories.tsx

```diff
@@ -13,53 +8,32 @@ export default {
   component: Tooltip,
 } as ComponentMeta<typeof Tooltip>;
 
-export const Default: Story<TooltipProps & UseTooltipProps> = (props) => {
-  const { getTriggerProps, getTooltipProps } = useTooltip(props);
+export const Default: Story<TooltipProps> = (props: TooltipProps) => (
+  <Tooltip {...props} content="I am a tooltip">
+    <Button>Hover</Button>
+  </Tooltip>
+);
 
-  const defaultProps = getTooltipProps({
-    title: "I am a tooltip",
-    status: "info",
-  });
-
-  return (
-    <>
-      <Button {...getTriggerProps<typeof Button>()}>Hover</Button>
-      <Tooltip {...defaultProps} />
-    </>
-  );
-};
-
-export const OpenTooltip: Story<TooltipProps & UseTooltipProps> = (props) => {
-  const { getTriggerProps, getTooltipProps } = useTooltip(props);
-
-  const openProps = getTooltipProps({
-    title: "I am a tooltip",
+export const OpenTooltip: Story<TooltipProps> = (props: TooltipProps) => {
+  const userProps: TooltipProps = {
     status: "info",
     open: true,
-  });
+    ...props,
+    content: props.content || "I am a tooltip",
+  };
 
   return (
-    <>
-      <Button {...getTriggerProps<typeof Button>()}>Hover</Button>
-      <Tooltip {...openProps} />
-    </>
+    <Tooltip {...userProps}>
+      <Button>Hover</Button>
+    </Tooltip>
   );
 };
 
-export const ScrollTooltip: Story<TooltipProps & UseTooltipProps> = (props) => {
+export const ScrollTooltip: Story<TooltipProps> = (props) => {
```

Comment:
in this example, while scrolling up and down the tooltip seems to swap if there is no more space and then follow the trigger up and down if you scroll. 
on left to right scroll, the tooltip seems to stay on screen even when the button is far scrolled out

Reply:
Should be better now